example_sns_init.yaml

```diff
@@ -112,9 +113,31 @@ Swap:
     restricted_countries:
         - AQ  # Antarctica
 
+    # Declare that each participant will get three SNS neurons in the following
+    # states:
+    # 1. Unlocked
+    # 2. Locked for 1 month
+    # 3. Locked for 2 months
     VestingSchedule:
-        events: 104
-        interval: 7 d
+        events: 3
+        interval: 1 month
+
+    # WARNING: Swap start_time works differently on mainnet and in testing.
+    #
+    # On mainnet:
+    # - Setting start_time to some value (e.g., 12:00 UTC) will allow the swap
```

Comment:
I wonder what happens if the proposal is accepted at 11.59 UTC: do the participants only have 1 min before the swap starts?

Reply:
I'm not 100% sure how this works. Maybe @DanielThurau could provide more info?

Reply:
No the algorithm will always add 24 hours at least, ad then find the next occurrence of `start_time`

Reply:
Thank you for the explanation. Should we include it in the comment?

Reply:
Thanks for confirming there's 24 hour minimum. However, I'm quite confused about your examples:

@mraszyk , did you mean to use `23.59 UTC` as an example? If so, according to this description:

> the algorithm will always add 24 hours at least, ad then find the next occurrence of start_time

the actual start time will be in 24 hours + 23:59, so it's two days minus 1 minute. Is this correct, @DanielThurau ?

Reply:
Let's take another start time which is not at noon/midnight to avoid confusion and let's use 24h format: if you set `start_time` to `18:15 UTC` in the yaml file and the proposal gets accepted and then _executed_ at `18:14 UTC` on Aug 11, I was wondering if the swap actually starts only one minute after the proposal was executed, i.e., at `18:15 UTC` on Aug 11, which wouldn't really fulfill the goal of getting the prospective participants enough time to prepare. But as @DanielThurau clarified, there's always at least a 24-hour window before the swap start. Hence, in the example from this message, the swap would actually start at `18:15` UTC on Aug 12.

@aterga 's confusion might come from the fact that the `start_time` is not an offset added to the 24-hour window, but rather an absolute time within a day (with respect to a particular time zone) when the swap is supposed to start. I believe that an absolute time within a day is preferrable to account for convenience within a preferred time zone.

Reply:
Thanks for the verbose explanations. Turns out there wasn't any confusion after all; I think we're all on the same page now.